exporter/clickhouseexporter/exporter_logs.go

```diff
@@ -117,12 +117,14 @@ func (e *logsExporter) pushLogsData(ctx context.Context, ld plog.Logs) error {
 	return err
 }
 
-func attributesToMap(attributes pcommon.Map) map[string]string {
-	m := make(map[string]string, attributes.Len())
-	attributes.Range(func(k string, v pcommon.Value) bool {
-		m[k] = v.AsString()
-		return true
-	})
+func attributesToMap(attributes ...pcommon.Map) map[string]string {
+	m := make(map[string]string)
+	for _, item := range attributes {
+		item.Range(func(k string, v pcommon.Value) bool {
+			m[k] = v.AsString()
```

Comment:
v.AsString() will return "" if the value is anything but a String. You can try `v.AsRaw()` then marshal the value to String.
```suggestion
			m[k] = v.AsRaw()
```

Reply:
> will return "" if the value is anything but a String.

its `v.Str()`?

Reply:
I don't remember. Please make sure to test with at least with one value that is not a string and that you get the right string when inserting.